gradle/java.gradle

```diff
@@ -16,8 +16,6 @@ plugins.withId('java') {
     apply plugin: 'maven'
 
     compileJava {
-        sourceCompatibility = "8"
```

Comment:
Why doesn't it work here, do you know?

Reply:
I am not a Gradle expert, but in some subproject this value is overridden or is not applied at all.
There are several questions on stackoverflow about this kind of problems with Gradle.
My fix seems to work and it makes sense to me